packages/fxa-payments-server/src/routes/Product/SubscriptionCreate/index.tsx

```diff
@@ -71,6 +94,10 @@ export const SubscriptionCreate = ({
     [selectedPlan]
   );
 
+  const { config } = useContext(AppContext);
+
+  const [paypalScriptLoaded, setPaypalScriptLoaded] = useState(false);
```

Comment:
It's weird that this is here but it's used above. I would move the whole `useEffect(() => {...});` from above to somewhere below this.